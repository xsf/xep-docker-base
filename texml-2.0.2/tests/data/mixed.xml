<TeXML>
In |text| mode.
<TeXML>Still in |text| mode</TeXML>
<TeXML mode="math">
In |math| mode: |&#x302;.
<TeXML mode="text">In |text| mode inside math mode</TeXML>
<TeXML>Still in |math| mode</TeXML>
</TeXML>
In |text| mode: |&#x302;.
The symbol can be only in math: &#x27F5;
<math>a&lt;b</math> <dmath>c&gt;d</dmath><spec cat="nl"/>
<dmath>
unicode mapping in math mode:
in text:  &#xa1;
in mixed: &#xE2;
in math:  &#x27F5;
none:     &#x777;
</dmath><spec cat="nl"/>
<TeXML>
unicode mapping in text mode:
in text:  &#xa1;
in mixed: &#xE2;
in math:  &#x27F5;
none:     &#x777;
</TeXML>
</TeXML>
