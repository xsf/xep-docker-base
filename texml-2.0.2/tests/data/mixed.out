In \textbar{}text\textbar{} mode. Still in \textbar{}text\textbar{} mode
In |math| mode: |\hat{}. In \textbar{}text\textbar{} mode inside math mode
Still in |math| mode In \textbar{}text\textbar{} mode: \textbar{}\^.
The symbol can be only in math: \ensuremath{\longleftarrow{}} $a<b$
$$c>d$$
$$ unicode mapping in math mode:
in text:  \ensuretext{\textexclamdown{}}
in mixed: \hat{a}
in math:  \longleftarrow{}
none:     \unicodechar{1911} $$
unicode mapping in text mode:
in text:  \textexclamdown{}
in mixed: \^{a}
in math:  \ensuremath{\longleftarrow{}}
none:     \unicodechar{1911}
